Source/Redemption/Private/BTTask_Fly.cpp

```diff
@@ -0,0 +1,40 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#include "BTTask_Fly.h"
+#include "Gargoyle.h"
+#include "PatrolPathIndicator.h"
+#include "AIController.h"
+#include "BehaviorTree/BlackboardComponent.h"
+#include "GameFramework/FloatingPawnMovement.h"
+
+#define GET_GARGOYLE Cast<AGargoyle>(OwnerComp.GetAIOwner()->GetPawn())
+#define GET_BLACKBOARD OwnerComp.GetBlackboardComponent()
```

Comment:
would it not be better to just save a local reference for something like this if you're making a define because it will be used often, then reduce the get calls altogether and do one and save it locally?

Reply:
I guess it would be better